### PR TITLE
Fix DustCover open/close

### PR DESCRIPTION
**Pull request by peter-englmaier**
Hi Jasem
Some recent refactoring work has broken my driver. The DustCap was not working anymore. This PR fixes the issue.
Regards,
Peter

### CODE REVIEW

File: drivers/focuser/alluna_tcs2.cpp [hunk 1]
```diff
@@ -844,33 +845,43 @@ bool AllunaTCS2::getDustCover()
...
+    getDustCover();
 
     // Cover already parked, nothing to do
     return IPS_OK;
 }
 
 IPState AllunaTCS2::UnParkCap()
 {
-    if (ParkCapSP[CAP_UNPARK].getState() == ISS_OFF)
+    LOGF_INFO("UnParkCap called, state is %d %d", ParkCapSP[CAP_PARK].getState(), ParkCapSP[CAP_UNPARK].getState());
```
Comment (knro) [inline drivers/focuser/alluna_tcs2.cpp:873]: Should be LOGF_DEBUG

File: drivers/focuser/alluna_tcs2.cpp [hunk 2]
```diff
@@ -844,33 +845,43 @@ bool AllunaTCS2::getDustCover()
...
     return true;
 }
 
 IPState AllunaTCS2::ParkCap()
 {
-    if (ParkCapSP[CAP_PARK].getState() == ISS_OFF)
+    if (ParkCapSP[CAP_PARK].getState() == ISS_ON)
     {
-        if (setDustCover()) // toggle state of dust cover
+        LOG_INFO("Toggle");
```
Comment (knro) [inline drivers/focuser/alluna_tcs2.cpp:857]: LOG_DEBUG

File: drivers/focuser/alluna_tcs2.cpp (hunk 1 above)
Comment (peter-englmaier) [reply drivers/focuser/alluna_tcs2.cpp:873]: Yes, I forgot to remove some debugging code.

Comment (peter-englmaier) [reply drivers/focuser/alluna_tcs2.cpp]: Fixed

File: drivers/focuser/alluna_tcs2.cpp (hunk 2 above)
Comment (peter-englmaier) [reply drivers/focuser/alluna_tcs2.cpp]: Fixed

**Comment by peter-englmaier** [comment]
I fixed the macos compile issue in ed3d1cca52df377fbb1df6cadafea3e9a1d25e3d, although it is not related to my code. If this should not be in this PR, I could move it to another PR.

**Pull request merged**